cli/azd/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
-## 0.2.0-beta.2 (Unreleased)
+## 0.2.0-beta.2 (2022-09-21)
+
+### Features Added
+
+- [[#616]](https://github.com/Azure/azure-dev/pull/616) Add `azd show --output json` for tooling use.
```

Comment:
@ellismg do we intend to announce this as a feature? I think it's fine, but we maybe should say something about it's stability? Like: "Add `azd show --output json` for tooling use. This currently is in limited preview support. Command output is not considered stable and may change in the future."

Reply:
Yeah - let's not put this in the changelog yet.  The command today doesn't show up in help, and is not intended for end users, just for editors like VS Code and VS.  When we are feeling better about it being used generally, we can add it to the changelog.

Thanks for looking out!